client/src/components/ratingsReviews/RatingsReviews.jsx

```diff
@@ -208,19 +200,24 @@ export default function RatingsReviews() {
   useEffect(() => {
     setTimeout(() => {
       setLoaded(true);
-    }, 400);
+    }, 600);
     // get review api data
     const getReviewApi = async () => {
       try {
         const res = await axios.get(
-          `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/reviews/?count=1000&product_id=${selectedProduct.id}`,
+          'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/reviews/',
           {
+            params: {
+              count: 1000,
+              product_id: selectedProduct.id
+            },
+
             headers: {
               Authorization: `${TOKEN}`,
             },
```

Comment:
Will be replaced with content type on server upload

Reply:
got it! Will see the replacement after yr PR.